lib/jekyll/commands/serve.rb

```diff
@@ -99,9 +103,9 @@ def webrick_opts(opts)
         # Recreate NondisclosureName under utf-8 circumstance
 
         private
-        def file_handler_opts
+        def file_handler_opts(opts)
           WEBrick::Config::FileHandler.merge({
-            :FancyIndexing     => true,
+            :FancyIndexing     => opts['show_dir_listing'] ? true : false,
```

Comment:
No @parkr, a "fancy" index in WEBrick-speak turns out to mean showing the contents of the directory when no index is present, which is what I'm trying to do. This is affirmed by the tests.

Anyway, @envygeeks made a good point about not changing behavior for a point release so I was planning to revert it anyway, so as not to hold up the merge. I thought of it as more of a bug fix based on what you were saying about parity with GHP. You decide.